src/schedule/dma_single_chan_domain.c

```diff
@@ -475,14 +475,14 @@ static void dma_single_chan_domain_set(struct ll_schedule_domain *domain,
 		goto out;
 
 	if (dma_domain->channel_changed) {
-		domain->last_tick = platform_timer_get(timer_get());
+		domain->next_tick = platform_timer_get(timer_get());
```

Comment:
I think you may need also to change the function comment.
```
/**
 * \brief Calculates domain's last tick.
```
PS: change next_tick to current time looks wired. Maybe add some inline comments

Reply:
> I think you may need also to change the function comment.
> 
> ```
> /**
>  * \brief Calculates domain's last tick.
> ```
Oh, yes, let me change it.
> 
> PS: change next_tick to current time looks wired. Maybe add some inline comments

Frankly speaking, I am not quite understand how the DMA driven work here, we have no topology which uses dma driven after CNL, what I change here is only replace the last_tick with next_tick.

@lgirdwood @mwasko @slawblauciak any comments about this dma_single_chan_domain change?

Reply:
We should schedule additional PR validation in DMA driven configuration, even if it is on CNL HW.